README.md

```diff
@@ -120,6 +120,68 @@ If you prefer live discussions, some of us also hang out in
 
 ## FAQ
 
+#### How do I configure alternative storage backends for LXD?
+
+LXD supports various storage backends; below are instructions on how to
+configure some of them. By default, we use a simple directory backed storage
+mechanism, but we recommend using ZFS for best results.
+
+###### ZFS
+
+First, you need to install the ZFS tooling. On Wily and above this is just:
+
+    sudo apt-get install zfsutils-linux
+
+ZFS has many different ways to procure a zpool, which is what you need to feed
+LXD. For example, if you have an extra block device laying around, you can
+just:
+
+    sudo zpool create lxd /dev/sdc6
+
+However, if you want to test things out on a laptop or don't have an extra disk
+laying around, ZFS has its own loopback driver and can be used directly on a
+(sparse) file. To do this, first create the sparse file:
+
+    sudo dd if=/dev/zero of=/var/lib/lxd.img bs=1k count=1 seek=100M
+
+then,
+
+    sudo zpool create lxd /var/lib/lxd.img
+
+Finally, whichever method you used to create your zpool, you need to tell LXD
+to use it:
+
+    lxc config set storage.zfs_pool_name lxd
+
+###### BTRFS
+
+The setup for btrfs is fairly simple, just mount /var/lib/lxd (or whatever your
+chosen `LXD_DIR` is) as a btrfs subvolume before you start LXD, and you're good
```

Comment:
We don't need for /var/lib/lxd to be a subvolume, it just must be on a btrfs filesystem.